connectors/spark/src/test/java/io/unitycatalog/spark/TableReadWriteTest.java

```diff
@@ -173,6 +173,32 @@ public void testCredentialDelta(String scheme) throws ApiException, IOException
     session.stop();
   }
 
+  @Test
+  public void testCredentialCreateDeltaTable() throws IOException {
```

Comment:
why isnt this test parameterized for s3, gs, abfs .. like the test below?

and